benchmarking/search_results.json

```diff
@@ -0,0 +1,2 @@
+{"commit_version": "9c929a86052c45483d2841c06c928bce805b92a4", "function_version": "0.0.1", "args": ["__unencodable_as_json__", "\"how to bring a cat from france to canada\""], "kwargs": {}, "function_name": "search", "start_time": "2023-10-20T18:05:37.477465", "finish_time": "2023-10-20T18:05:37.955207", "return_value": [{"search": [{"id": "e9a64ea3-c7c4-4f0e-bd49-c4e1d9de1a7f", "url": "https://inspection.canada.ca/importing-food-plants-or-animals/pets/eng/1326600389775/1326600500578", "score": 3.017821829567311, "title": "Bringing animals to Canada: Importing and travelling with pets - Canadian Food Inspection Agency", "scores": {"current": 1, "recency": 0.8708793589157114, "traffic": 0.8188254189764937, "similarity": 0.8272663498460967, "typicality": 0.004253509145044662}, "content": "Bringing animals to Canada: Importing and travelling with pets If you are travelling with a pet or planning to import an animal to Canada, you will need the right paperwork at the border to meet Canada's import requirements\n If you don't, you risk experiencing delays at the border and your animal may not be allowed into Canada\n Canada has specific import requirements in place to avoid introducing animal diseases to protect its people, plants and animals", "query_id": "3bf1f7e9-7219-410a-addb-0f86b451c87a", "subtitle": "Bringing animals to Canada: Importing and travelling with pets", "last_updated": "2022-10-20", "tokens_count": 84}]}]}
```

Comment:
__unencodable_as_json__ is because the first argument is an object (cursor)

You can fix by 

1) subclassing JSONEncoder class:

https://github.com/alubbock/microbench/blob/474f7439a7929d8538a4ba4d009a342a063ec06b/microbench/__init__.py#L47

2) passing this extended JSONEncoder to the instantiation of MicroBench:

https://github.com/alubbock/microbench/blob/474f7439a7929d8538a4ba4d009a342a063ec06b/microbench/__init__.py#L70

Reply:
turns out it was trying to put the cursor in the args, in my opinion it shouldn't appear since it includes some of our environment variables.

I changed he file to put the benchmark on a different function that would only use the query as args